### ISSUE TITLE
Options for 'status assigned to games played for the first time'

### ISSUE THREAD

**Issue by chocolatechipcats** [JosefNemec/Playnite#3379]
### Feature description

Related: #3377 

Currently, the 'status assigned to games played for the first time' only applies if the status matches the 'default status assigned to newly added games.' 

If users have an additional plan to play/want to play/backlog completion status, these will not automatically change.

1. Apply the status to _any_ played game with a playtime and/or playcount of 0. (How would this interact with games with imported playtimes?)
2. Have the option to assign additional 'unplayed' statuses to be changed upon play. 

I personally prefer option 2, mostly because I can also put my 'on hold' status there too.

Thanks in advance.

**Comment by woutercools** [comment]
> I personally prefer option 2, mostly because I can also put my 'on hold' status there too.

+1

**Comment by woutercools** [comment]
I see version 10.15 adds this option.
I just can't see where I can enable/disable it.

**Comment by chocolatechipcats** [comment]
No, the new option is for something completely different.

**Comment by woutercools** [comment]
> Distinct options to remove or keep completion untouched after playing a game for the first time

I don't know but isn't it exactly what "options for 'status assigned to games played for the first time" mean, no?

**Comment by chocolatechipcats** [comment]
No. I suggested adding additional circumstances for the status to change. This just adds some extra statuses to change to.

**Comment by JosefNemec** [comment]
Yes, the way Playnite currently detects that a game was played for the first time hasn't changed with that update.

**Comment by woutercools** [comment]
> Yes, the way Playnite currently detects that a game was played for the first time hasn't changed with that update.

Is the new option related to issue [#3377](https://github.com/JosefNemec/Playnite/issues/3377)?

**Comment by Sprabuni** [comment]
+1
Based on the wording of the setting, I would have expected this to already overwrite other statuses, so I want to see something like this implemented too. My setup assigns different status to newly imported games (none), games I don't want to play (not played) and games I do want to play (plan to play), so it's impossible for me to use this setting currently.
